lib/checkPullRequestBranch.js

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const DISSALLOWED_BRANCH_PREFIXES = ['develop', 'release-', 'test-'];
+const DISSALLOWED_BRANCH_PREFIXES = ['develop-', 'release-', 'test-'];
```

Comment:
Why this change?

Reply:
Because now the invalid names include branches with prefixes. So this differentiates between develop branch and develop prefixed branch.